spec/factories/clearance.rb

```diff
@@ -4,7 +4,18 @@
   end
 
   factory :user do
-    email
+    email { generate(:email) }
     password { 'password' }
   end
 end
+
+FactoryBot.define do
+  sequence :name do |n|
+    "chicken#{n}"
+  end
+
+  factory :recipe do
+    name { :name }
+    user { :user }
+  end
+end
```

Comment:
I'd move this to it's own file. It technically has nothing to do with `clearance` and I usually see each model with its own factory file.  So this would go in **spec/factories/recipe.rb**.  Honestly, you could probably rename this file as well to be **spec/factories/user.rb**